### PR TITLE
media-radio/svxlink: add 24.02

### PR DESCRIPTION
**Pull request by hamarituc**
version bump

Adds conditional dependencies for RTL2832- and `libgpiod`-support.

---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: media-radio/svxlink/svxlink-24.02.ebuild [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (mm1ke) [inline media-radio/svxlink/svxlink-24.02.ebuild:4]: should be `EAPI=8`

File: media-radio/svxlink/svxlink-24.02.ebuild [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+CMAKE_MAKEFILE_GENERATOR=emake
+
+inherit cmake flag-o-matic systemd
+
+CMAKE_USE_DIR="${S}/src"
+
+DESCRIPTION="Multi Purpose Voice Services System, including Qtel for EchoLink"
+HOMEPAGE="http://www.svxlink.org"
```
Comment (mm1ke) [inline media-radio/svxlink/svxlink-24.02.ebuild:13]:
```suggestion
HOMEPAGE="https://www.svxlink.org"
```

File: media-radio/svxlink/svxlink-24.02.ebuild [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+SRC_URI="https://github.com/sm0svx/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-2.1"
```
Comment (mm1ke) [inline media-radio/svxlink/svxlink-24.02.ebuild:16]: I've checked some source files and there were `GPL-2+`. I don't know about the LGPL, maybe you know which files are licenses under LGPL and maybe they are too under `LGPL-2.1+` ?

**Comment by hamarituc** [comment]
Was updated by the maintainer in the meantime.

**Pull request closed without merging**